src/linear/updater_gpu_coordinate.cu

```diff
@@ -53,17 +53,13 @@ class DeviceShard {
     // iterate through columns
     for (auto fidx = 0; fidx < batch.Size(); fidx++) {
       common::Span<Entry const> col = batch[fidx];
-      auto cmp = [](Entry e1, Entry e2) {
-        return e1.index < e2.index;
-      };
-      auto column_begin =
-          std::lower_bound(col.cbegin(), col.cend(),
-                           xgboost::Entry(row_begin, 0.0f), cmp);
-      auto column_end =
-          std::lower_bound(col.cbegin(), col.cend(),
-                           xgboost::Entry(row_end, 0.0f), cmp);
-      column_segments.emplace_back(
-          std::make_pair(column_begin - col.cbegin(), column_end - col.cbegin()));
+      auto cmp = [](Entry e1, Entry e2) { return e1.index < e2.index; };
```

Comment:
These formatting changes make the PR confusing. It's not immediately obvious if the changes are only formatting and this file will be included in the diff even when nothing has changed relating to the commit message.